LICENSE

```diff
@@ -36,7 +39,7 @@ exhaustive, and do not form part of our licenses.
      material not subject to the license. This includes other CC-
      licensed material, or material used under an exception or
      limitation to copyright. More considerations for licensors:
- wiki.creativecommons.org/Considerations_for_licensors
+    wiki.creativecommons.org/Considerations_for_licensors
```

Comment:
This (and the other link 17 lines below) still look misindented, although it could be github.  (Could also be spaces versus tabs?)  I'm also curious why this changed at all -- was this license previously mis-pasted from the source?

Reply:
Yeah, it looks bad to me too. It's copied verbatim from https://creativecommons.org/licenses/by/4.0/legalcode.txt, which I suppose has changed since it was copied the first time. I'm guessing it's derived from some other source.

One thing I've been tempted to do, suggested by @annevk, is to trim the pre- and post-amble to the CC-BY license, which aren't part of the license. I'll push a commit like that, which removes this bit too and means we don't need to be bothered by its indentation.

Reply:
(Now done, but leaving this conversation unresolved for visibility.)